pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -572,17 +576,19 @@ public class ServiceConfiguration implements PulsarConfiguration {
     @FieldContext(
         category = CATEGORY_POLICIES,
         dynamic = true,
-        doc = "Enable the deletion of inactive topics.\n"
-        + "If only enable this option, will not clean the metadata of partitioned topic."
+        doc = """
+                Enable the deletion of inactive topics.
+                If only enable this option, will not clean the metadata of partitioned topic."""
     )
     private boolean brokerDeleteInactiveTopicsEnabled = true;
     @FieldContext(
             category = CATEGORY_POLICIES,
             dynamic = true,
-            doc = "Metadata of inactive partitioned topic will not be automatically cleaned up by default.\n"
-            + "Note: If `allowAutoTopicCreation` and this option are enabled at the same time,\n"
-            + "it may appear that a partitioned topic has just been deleted but is automatically created as a "
-                    + "non-partitioned topic."
+            doc = """
+                    Metadata of inactive partitioned topic will not be automatically cleaned up by default.
+                    Note: If `allowAutoTopicCreation` and this option are enabled at the same time, \
+                    it may appear that a partitioned topic has just been deleted but is automatically created as a \
+                    non-partitioned topic."""
```

Comment:
```suggestion
            doc = """
                    Metadata of inactive partitioned topic will not be automatically cleaned up by default.
                    Note: If `allowAutoTopicCreation` and this option are enabled at the same time,
                    it may appear that a partitioned topic has just been deleted but is automatically created as a
                    non-partitioned topic."""
```